test/e2e/windows/host_process.go

```diff
@@ -657,7 +657,8 @@ var _ = sigDescribe(feature.WindowsHostProcessContainers, "[MinimumKubeletVersio
 
 		ginkgo.By("Waiting for the pod to start running")
 		timeout := 3 * time.Minute
-		e2epod.WaitForPodsRunningReady(ctx, f.ClientSet, f.Namespace.Name, 1, 0, timeout)
+		err = e2epod.WaitForPodsRunningReady(ctx, f.ClientSet, f.Namespace.Name, 1, timeout)
+		framework.ExpectNoError(err)
```

Comment:
One of the CI checks made me add these checks for errors

Reply:
And for a good reason! The test was clearly broken without this error check.

Reply:
Side note: in some cases I found tests that were no longer passing when adding missing error checking.

Reply:
Agreed!

For the record, the fact that "failing to properly detect an error in this function constitutes the test that calls it being broken" is the reason that I thought it was important to fix the logic at the end of `WaitForAlmostAllPodsReady`. The logic for when the function returns an error is clearly busted, which is at least as bad as failing to check for errors.